lib/modules/dosomething/dosomething_kudos/includes/KudosController.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+class KudosController extends EntityAPIController {
+
+  public function __construct() {
+    parent::__construct('kudos');
+  }
+
+  /**
+   * Overrides create() method in EntityAPIController
+   *
+   * @param array $values
+   * @return bool
+   *
+   * @throws Exception
+   */
+  public function create(array $values = []) {
+    $values += ['is_new' => TRUE];
+
+    $kudos = new Kudos($values);
+
+    $record = $this->save($kudos);
+
+    if ($record) {
+      return TRUE;
+    }
+
+    return FALSE;
+  }
+
+
+  /**
+   * Overrides delete() method in EntityAPIController.
+   * Provides better feedback for API.
```

Comment:
Can you also specify which lines the override is for here too?  I'm just wondering what the specifics are for why there aren't any calls to the parent functions in these...